### PR TITLE
Move `electrical` package to `metrics.electrical`

### PR DESCRIPTION
**Pull request by leandro-lucarella-frequenz**
Also fix the `components` package name and add a missing `syntax` to the `metrics` package.

### CODE REVIEW

File: proto/frequenz/api/common/metrics/electrical.proto
Comment (tiyash-basu-frequenz) [inline proto/frequenz/api/common/metrics/electrical.proto:11]: For this packaging, the `electrical.proto` file needs to be in a metrics directory.

I'd suggest
1. renaming [`metrics.proto`](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.x.x/proto/frequenz/api/common/metrics.proto) to `metric.proto`, and changing the package name there to `frequenz.api.common.metric`.
2. putting `electrical.proto` in a directory here named `metrics`.

**Review by tiyash-basu-frequenz — CHANGES_REQUESTED** [review]

File: proto/frequenz/api/common/metrics/electrical.proto
Comment (leandro-lucarella-frequenz) [reply proto/frequenz/api/common/metrics/electrical.proto:11]: I don't get it, it is inside the `metrics` directory :thinking:

Comment (tiyash-basu-frequenz) [reply proto/frequenz/api/common/metrics/electrical.proto:11]: I see. This is what I was referring to in point 2.

Comment (tiyash-basu-frequenz) [reply proto/frequenz/api/common/metrics/electrical.proto:11]: Actually, point 1 would mean that we use `metric::Metric` and `metrics::electrical::Dc/Ac`.
`metrics::Metric` would make be more consistent here.

There is nothing to be done for this comment.

I can approve once the CI passes.

Comment (leandro-lucarella-frequenz) [reply proto/frequenz/api/common/metrics/electrical.proto:11]: The CI doesn't pass for unrelated reasons (the main branch is also failing). I will create a separate PR with the fixes for the CI.

**Comment by leandro-lucarella-frequenz** [comment]
Moved all fixes to:

* https://github.com/frequenz-floss/frequenz-api-common/pull/5

This will pass the CI after that PR is merged and this is rebased on top of it.

File: proto/frequenz/api/common/metrics/electrical.proto
Comment (leandro-lucarella-frequenz) [reply proto/frequenz/api/common/metrics/electrical.proto:11]: I also enabled branch protection to require test to pass.

**Comment by stefan-brus-frequenz** [comment]
Not sure why protolint is complaining here. I'll try re-running it.

**Comment by stefan-brus-frequenz** [comment]
> Moved all fixes to:
> 
> * [Fix CI and migrate to repo-config #5](https://github.com/frequenz-floss/frequenz-api-common/pull/5)
> 
> This will pass the CI after that PR is merged and this is rebased on top of it.

Ah, just saw this.

**Comment by leandro-lucarella-frequenz** [comment]
Rebased on top of the current base branch and enabled auto-merge.

**Review by tiyash-basu-frequenz — APPROVED** [review]

**Pull request merged**
